### PR TITLE
Updated the Berksfile source to point to supermarket.chef.io

### LINKED ISSUE #19: Berksfile - Switch "site:opscode" to "source:supermarket"
The Berksfile currently points the source to be **site :opscode** instead of **source :'https://supermarket.chef.io'** or some other index server.  Please update to point to the correct location.

### PR DESCRIPTION
**Pull request by dolendupes**
Fixes #19 by modifying the Berksfile source to "https://supermarket.chef.io"

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by jtimberman** [comment]
:+1: obvious fix imo

**Comment by dolendupes** [comment]
Additional documentation 'README' commits to simplify the process of setting up and verifying the push job client.

File: README.md [hunk 1]
```diff
@@ -131,6 +141,35 @@ manager with remote repositories. The URL will be used (with the
...
+
+The connection information for the push server is established when the
+push server is installed and the Chef Server is reconfigured.  In the case
+the Chef Server is not providing the correct push server configuration, 
+please verify hostnames are correct and that both the push server and 
+Chef Server have been reconfigured.
+
+# Verify Push Jobs Client Connection
+
+If the push-jboss client has been successfully installed on a node, the
```
Comment (manderson26) [inline README.md]: pushy-jobs client? I like jboss though :-)

**Comment by manderson26** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (dolendupes) [inline README.md]: Oops.  My past is sneaking up on me.

**Pull request closed without merging**